sdk/identity/identity/src/msal/nodeFlows/msalClient.ts

```diff
@@ -476,7 +484,7 @@ To work with multiple accounts for the same Client ID and Tenant ID, please prov
    */
   function calculateRequestAuthority(options?: GetTokenOptions): string | undefined {
     if (options?.tenantId) {
-      return getAuthority(options.tenantId, getAuthorityHost(createMsalClientOptions));
+      return getAuthority(options.tenantId, createMsalClientOptions.authorityHost);
```

Comment:
Was this intentional? I believe we need to use `getAuthorityHost`

Reply:
Accidental